staging_update_configs_playbook.yml

```diff
@@ -9,13 +9,9 @@
       - host_vars/staging.usegalaxy.org.au.yml
       - secret_group_vars/ubuntu_maintenance_key
   handlers:
-    - name: restart handlers
-      become: True
-      become_user: galaxy
-      command: "{{ galaxy_venv_dir }}/bin/galaxyctl restart handler_0 handler_1 handler_2"
-      environment:
-        GRAVITY_STATE_DIR: "{{ galaxy_gravity_state_dir }}"
-        GALAXY_CONFIG_FILE: "{{ galaxy_config_file }}"
+    - name: galaxy gravity restart handlers
```

Comment:
```suggestion
    - name: restart handlers
```

Reply:
I don't know if we can have a central handler for multiple playbooks, worth checking out but not today. For the production playbooks we can now use the defaults from the galaxyproject.galaxy role. These update playbooks do not use the galaxyproject.galaxy role.  There are three of them: one for dev/staging/production.

Reply:
I think that's an implementation detail and should ideally not be relevant to the core task of restarting handlers, so just thought I'd flag it. Fine with leaving it as is also.